jerry-ext/include/jerryscript-ext/module.h

```diff
@@ -29,9 +29,9 @@ typedef jerry_value_t (*jerryx_native_module_on_resolve_t) (void);
  */
 typedef struct jerryx_native_module_t
 {
-  const jerry_char_t *name_p; /**< name of the module */
-  const jerryx_native_module_on_resolve_t on_resolve; /**< function that returns a new instance of the module */
-  struct jerryx_native_module_t *next_p; /**< pointer to next module in the list */
+  const jerry_char_t *name_p;
+  const jerryx_native_module_on_resolve_t on_resolve;
+  struct jerryx_native_module_t *next_p;
```

Comment:
Why did you remove the comments from here?

Reply:
@zherczeg [said](https://github.com/jerryscript-project/jerryscript/pull/2013#discussion_r140710783) we didn't add such `/**<` comments to the header files. Or did I misinterpret?

Reply:
Yes, I think there was a misunderstanding here. We do not add comments to declarations, but only to definitions. The type definitions are usually in the header files, so we must put the comments there. Please don't remove these comments.

Reply:
Sorry, I mean only for function arguments, not for types.